src/System.Data.SqlClient/Directory.Build.props

```diff
@@ -4,6 +4,9 @@
     <!-- Must be kept in sync with pkg\Microsoft.Windows.Compatibility\Microsoft.Windows.Compatibility.pkgproj -->
     <PackageVersion>4.8.1</PackageVersion>
     <AssemblyVersion>4.6.1.1</AssemblyVersion>
+    <!-- Downgrade the Assembly Version to match RTM in order to have System.Data shim to still 
+    typeforward to RTM version instead of servicing version -->
```

Comment:
What if we put this same condition in all oob libs that were the target of shims?  Even if they haven’t bumped the version yet in servicing?

Reply:
We could do that, but that might have unintended impacts, like for example it may affect other OOB packages that do provide netcoreapp assets and that reference the assembly version that we pinned. Another risk I see with adding a bunch of netcoreapp configurations everywhere, is that we will start preferring those for testing during our vertical build (assets that won't ship anywhere), and we will drop coverage of netstandard assets on test runs that do get published in nuget packages.

Reply:
Is there a problem if they reference the RTM version?  We don’t change API in servicing. What if this was constrained only to reference assembly?

Reply:
I see what you mean about leaking elsewhere.